ton_client/src/tests/mod.rs

```diff
@@ -288,7 +288,7 @@ impl TestClient {
     }
 
     pub fn node_se() -> bool {
-        std::env::var("USE_NODE_SE").unwrap_or(DEFAULT_USE_NODE_SE.to_owned()) == "true".to_owned()
+        std::env::var("USE_SE").unwrap_or(DEFAULT_USE_SE.to_owned()) == "true".to_owned()
```

Comment:
don't rename

Reply:
there is no such product any more, now it is tonos se or just SE